packages/atomic/scripts/generate-component-templates/component.ts.hbs

```diff
@@ -1,6 +1,6 @@
-import {withTailwindStyles} from '@/src/decorators/with-tailwind-styles.js';
```

Comment:
Also out of scope.

As I tested the generation script with the changes I had made, I realized that the imports were not correctly ordered and that there were some TS errors.